examples/mobile/Makefile

```diff
@@ -2,7 +2,7 @@
 
 DEVICE = ${DEVICE_ID}
 ifndef DEVICE_ID
-	DEVICE=$(shell xcrun simctl list devices 'iOS' | grep -v 'unavailable' | grep -v '^--' | grep -v '==' | head -n 1 | grep -E -o -i "([0-9a-f]{8}-([0-9a-f]{4}-){3}[0-9a-f]{12})")
+	DEVICE=$(shell xcrun simctl list devices 'iOS' | grep -v 'Shutdown' | grep -v '^--' | grep -v '==' | head -n 1 | grep -E -o -i "([0-9a-f]{8}-([0-9a-f]{4}-){3}[0-9a-f]{12})")
```

Comment:
seems unrelated

Reply:
I  modified this to be able to test the iOS app. I'll add both to the grep.